.github/workflows/test-action-tuf.yaml

```diff
@@ -21,15 +21,15 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-        - v1.23.x
-        - v1.24.x
-        - v1.25.x
+          - v1.23.x
```

Comment:
should we drop 1.23? 
its eol on feb 28

Reply:
Yeah, well, this is a bit tricky because I know some folks have asked fairly recently for policy-controller support on something like 1.21, 1.22 so keeping testing as long as we can is something that I think would be nice. If there are breaking changes that would require making changes, then we can revisit them on case-by-case basis. Does that make sense? But I don't feel super strongly, just think it would be nice :)

Reply:
lol, it's a great question that we should have an answer to. Tricky bit is that indeed for us to run on older versions because we're relying on knative we will need to use the ENV variable to say it's ok to run. So 🤷